fv3gfs/physics/global_config.py

```diff
@@ -13,7 +13,7 @@
 FIELD_FLTI = gtscript.Field[gtscript.I, DTYPE_FLT]
 FIELD_FLTJ = gtscript.Field[gtscript.J, DTYPE_FLT]
```

Comment:
This file should be split into three: global_config.py, types.py/typing.py, and constants.py. We should plan to remove the global config component soon, as we're doing in the dycore.

Reply:
Does it make sense to keep separate config, types, and constant file in physics? Should this information be moved up to the integration directory?